pom.xml

```diff
@@ -35,9 +35,8 @@
     <jenkins.version>2.426.3</jenkins.version>
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
     <byte-buddy.version>1.14.9</byte-buddy.version>
-    <node.version>10.13.0</node.version>
-    <npm.version>6.14.4</npm.version>
-    <!-- TODO: Remove this and related dependencyManagement entries once it is included in the BOM. -->
+    <node.version>20.12.1</node.version>
+    <npm.version>10.5.0</npm.version>
```

Comment:
```suggestion
    <npm.version>10.5.0</npm.version>
    <!-- TODO: Remove this and related dependencyManagement entries once it is included in the BOM. -->
```
since that relates to https://github.com/jenkinsci/blueocean-plugin/blob/26e9c44bb0af15754d8fd920f776394a99d505b7/pom.xml#L40